README.md

```diff
@@ -66,7 +66,7 @@ local result, err = crud.insert_object(space_name, object, opts)
 
 where:
 
-* `space_name` (`string`) - name of the space to insert an object
+* `space_name` (`string|number`) - name or numerical id of the space to insert an object
```

Comment:
Is it tested anyhow?

Won't it be a problem because on different replicasets the same space could have different id?

Reply:
No actually, I have reverted these changes.